### ISSUE TITLE
[A11y]The card can be closed by esc key when focusing on the frame but cannot be closed  by esc key when focusing on the field in frame.

### ISSUE THREAD

**Issue by wanghe2015** [MicrosoftDocs/msteams-docs#11501]
### Steps to reproduce

Hi team,
We are doing A11y test for the frame card in Teams. We found when focusing on the frame, the card can be closed by esc key. But if it is not focusing on the frame, for example, it is focusing on the field in the frame, esc key cannot close the card.

### Expected behavior

Is it working by design? - When focusing on the frame, esc key can close the card. But when focusing on the field in the frame, esc key cannot close the card.

### Actual behavior

When focusing on the frame, esc key can close the card. But when focusing on the field in the frame, esc key cannot close the card.

<img width="434" alt="image" src="https://github.com/user-attachments/assets/303df8cf-2669-40e5-9da8-668639fe310d">

<img width="480" alt="image" src="https://github.com/user-attachments/assets/53437cf3-6e4b-4995-846c-a127b387cf9b">

**Comment by Nivedipa-MSFT** [comment]
@wanghe2015 - Thanks for reporting your issue. We will test this and update you soon.

**Comment by Meghana-MSFT** [comment]
We are able to repro this using an ME based action app. We checked using Polls app.

**Focus on the Frame: Esc key closes the popup:**
![image](https://github.com/user-attachments/assets/d531462c-5b00-4c5b-aa17-b27dc09f6617)

**Focus on elements inside frame: Esc key doesn't close the popup**
![image](https://github.com/user-attachments/assets/89fb2e39-1b01-4759-b358-3eefb73d8a56)



 We raised a bug for this, we will keep you posted on the updates. Thank you.
